rust/decentraland-godot-lib/src/dcl/js/js_modules/main.js

```diff
@@ -126,5 +126,4 @@ globalThis.performance = {
 
 Deno.core.ops.op_set_promise_reject_callback((type, promise, reason) => {
     console.error('Unhandled promise: ', reason)
-    Deno.core.ops.op_promise_reject();
```

Comment:
It's a test function that isn't available in deno_core, so the scene breaks because you can't call undefined.
The test function is used to count how many rejections are

![image](https://github.com/decentraland/godot-explorer/assets/8042536/25de17d1-376e-45d7-aae2-ae15ccb8c969)